modules/administration-guide/partials/proc_migration-to-devworkspace-engine.adoc

```diff
@@ -0,0 +1,210 @@
+
+[id="migration-to-devworkspace-engine_{context}"]
+= Migration to {devworkspace} engine.
+
+This procedure describes how to migration to {devworkspace} engine using https://docs.openshift.com/container-platform/latest/operators/understanding/olm/olm-understanding-olm.html[OLM] to support the Devfile 2.0.0 file format and mentions how to do so on existing instances.
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+* An instance of {prod-short} running in OpenShift 4 cluster.
+* OpenShift OAuth is enabled. See xref:configuring-openshift-oauth.adoc[].
+
+.Procedure
+
+. Save and push changes back to the Git repositories for all running workspaces of the {prod-short} instance.
+
+. Stop all workspaces in the {prod-short} instance.
```

Comment:
How can I, as an admin of Che, do that?

Reply:
Good question. I will replace on something: All workspaces must be stopped and changes pushed back to Git repositories